### PR TITLE
Update docs for v9

### PR DESCRIPTION
**Pull request by AyushBherwani1998**
## Motivation and Context
Updates the Web Modal PnP SDK documentation for the v9 release.

Jira Link: https://toruslabs.atlassian.net/browse/DEVREL-972?atlOrigin=eyJpIjoiMGI1YmI5ZTVkMDlhNGU4Njk0YWFjYzUyZTc3ZTE5MGMiLCJwIjoiaiJ9

## Description

Updates doc for Modal, Wallet Services, and Default Adapters. 

## How has this been tested?

The docs have build and tested locally

## Screenshots (if appropriate):
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project. (run lint)
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/sdk/pnp/web/adapters/auth.mdx [hunk 1]
```diff
@@ -289,23 +282,190 @@ instance. This function takes the below-mentioned parameters as well as
...
-| ------------------ | ------------------------ |
-| `clientId?`        | `string`                 |
-| `sessionTime?`     | `number`                 |
-| `chainConfig?`     | `CustomChainConfig`      |
-| `web3AuthNetwork?` | `OPENLOGIN_NETWORK_TYPE` |
+`AuthOptions`
+
+| Parameter               | type                       |
+| ----------------------- | -------------------------- |
+| `clientId`              | `string`                   |
```
Comment (shahbaz17) [inline docs/sdk/pnp/web/adapters/auth.mdx:289]: write description and include type in it

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/sdk/pnp/web/adapters/auth.mdx (hunk 1 above)
Comment (AyushBherwani1998) [reply docs/sdk/pnp/web/adapters/auth.mdx:289]: @ihsraham can you check?

Comment (ihsraham) [reply docs/sdk/pnp/web/adapters/auth.mdx:289]: Will add it

**Review by yashovardhan — APPROVED** [review]

**Review by yashovardhan — APPROVED** [review]
Left some comments for migration guide. Rest everything is good to go.

**Pull request merged**